examples/installation/ios/swift-4.0

```diff
@@ -1 +0,0 @@
-swift-3.1
```

Comment:
They were all files when I checked out that version. So I just policed them all to be symlinks.